SocketRocket/SRSecurityPolicy.h

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2012 Square Inc.
+// Portions Copyright (c) 2016-present, Facebook, Inc.
+//
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol SRSecurityPolicy <NSObject>
+
+- (BOOL)evaluateServerTrust:(SecTrustRef)serverTrust
```

Comment:
Nit: Without import for Security.framework - looks like importing this header will break compilation.
TL;DR Add `#import <Security/Security.h>` to the imports of this header.